packages/webpack/src/plugin.js

```diff
@@ -139,6 +145,16 @@ class LavaMoatPlugin {
           )
         }
 
+        // =================================================================
+        const policyGenerator = createPolicyGenerator({
+          policyFromOptions: options.policy,
+          enabled: !!options.generatePolicy,
```

Comment:
(style)

```suggestion
          enabled: Boolean(options.generatePolicy),
```

Reply:
I like `!!` better. If prettier didn't fix it, maybe it's fine?

Reply:
Personally I'd consider changing generatePolicy to shouldGeneratePolicy to indicate it's a bool but mainly to tell it apart from other instances of generatePolicy function in the codebase which are two different things (I was confused by that browsing through the code)